docs/python-grader/Makefile

```diff
@@ -2,7 +2,7 @@
 # Your mileage may vary if you are running this locally
 
 SPHINXOPTS    ?= -c sphinx-config
-SPHINXBUILD   ?= python3 /usr/local/lib64/python3.6/site-packages/sphinx/cmd/build.py
+SPHINXBUILD   ?= sphinx-build
```

Comment:
On my machine, this was just available on my `PATH`. I'm hoping that'll generally be true?

Reply:
This should be true in general